packages/rps/src/redux/message-service/firebase-inbox-listener.ts

```diff
@@ -3,13 +3,17 @@ import {take, call} from 'redux-saga/effects';
 import {reduxSagaFirebase} from '../../gateways/firebase';
 import {RPSChannelClient} from '../../utils/rps-channel-client';
 import {Message} from '@statechannels/channel-client';
+import {buffers} from 'redux-saga';
 
 export function* firebaseInboxListener(client: RPSChannelClient) {
   const address: string = yield call([client, 'getAddress']);
+  // TODO: Redux saga firebase accepts a buffer but the type definition hasn't been updated in the latest version (0.15.0)
+  // so we just cast as any for now
```

Comment:
Is there an issue for this on their repo? I think I'm the one that removed this argument

Reply:
I've opened this https://github.com/n6g7/redux-saga-firebase/issues/567